ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml

```diff
@@ -1225,27 +1225,33 @@ presubmits:
     branches:
     - ^master$
     - ^master-
-    cluster: build05
+    cluster: build03
     context: ci/prow/ocp-4-6-operator-e2e-tests
     decorate: true
     decoration_config:
       skip_cloning: true
       timeout: 5h0m0s
     labels:
+      ci-operator.openshift.io/cloud: aws
+      ci-operator.openshift.io/cloud-cluster-profile: aws
+      ci-operator.openshift.io/variant: ocp-4-6
       ci.openshift.io/generator: prowgen
-      job-release: "4.13"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-stackrox-stackrox-master-ocp-4-6-operator-e2e-tests
     optional: true
     rerun_command: /test ocp-4-6-operator-e2e-tests
+    run_if_changed: ^((generated|image|operator|pkg|make|deploy|scripts)/.*|Makefile|status\.sh|EXPECTED_GO_VERSION)
```

Comment:
you may need to manually delete this after removing in config and running `make update`.